Automattic/simplenote-ios
# Switch to appcenter

### Fix
This PR updates the app to use the new MSAppCenter SDK instead of the HockeyApp one. AppCenter should be able to interact with the old SDK for a while, but since we're migrating, it  makes sense to jump on the new one now.

### Test
1. Pull the latest version of the `.mobile-secrets` repository.
2. Build the internal version of the app and make sure the build completes without errors. 

### Release
These changes do not require release notes.
